lib/lib_rss.php

```diff
@@ -48,7 +48,31 @@ function classAutoloader($class) {
 	} elseif (strpos($class, 'CssXPath') !== false) {
 		include(LIB_PATH . '/CssXPath/' . basename(str_replace('\\', '/', $class)) . '.php');
 	} elseif (strpos($class, 'PHPMailer') === 0) {
-		include(LIB_PATH . '/' . str_replace('\\', '/', $class) . '.php');
+		// project-specific namespace prefix
+		$prefix = 'PHPMailer\\';
+
+		// base directory for the namespace prefix
+		$base_dir = LIB_PATH . '/phpmailer/phpmailer/src/';
+
+		// does the class use the namespace prefix?
+		$len = strlen($prefix);
+		if (strncmp($prefix, $class, $len) !== 0) {
+			// no, move to the next registered autoloader
+			return;
+		}
+
+		// get the relative class name
+		$relative_class = substr($class, $len);
+
+		// replace the namespace prefix with the base directory, replace namespace
+		// separators with directory separators in the relative class name, append
+		// with .php
+		$file = $base_dir . str_replace('\\', '/', $relative_class) . '.php';
+
+		// if the file exists, require it
+		if (file_exists($file)) {
```

Comment:
What is the added value of `file_exists()` (which is an expensive IO call)? `require` will properly fail if the file does not exist.